### DISCUSSION TITLE
Timelapse and KAMP

### DISCUSSION THREAD

**Discussion by golia28** [Guilouz/Creality-Helper-Script-Wiki#428] [General]
<img width="853" alt="Screenshot 2024-03-13 alle 12 00 23" src="https://github.com/Guilouz/Creality-K1-Series/assets/31170900/8a8eb0ee-b23a-4bca-b36c-6d76b283bea3">
Hi, I recently installed timelapse from your helper, the configuration is a fully klipper k1, without Creality software.
I had some trouble configuring it and now in the Timelapse section on Fluidd i see a .zip file with the jpg only, no video there.
The other thing is that now I'm having an error message regarding KAMP, like see in the screenshot attached. Otherwise KAMP is still working great.

Thanks

**Comment by Guilouz** [comment]
If you read the changelog you will see that KAMP repo have been removed because KAMP it now included in new script.

**Reply by golia28** [reply]
Ok, what about the timelapse?

**Reply by Guilouz** [reply]
Read the wiki to how configure timelapse

**Reply by golia28** [reply]
I followed the instruction step by step, and it seems not so difficult.  Everything seems installed and configured like in the helper wiki, orcaslicer is configured too.
I have the timelapse page in fluidd but can get only the images, no video. What I'm missing?
